bin/GameIndex.dbf

```diff
@@ -16825,7 +16846,7 @@ Region = PAL-M5
 Serial = SLES-54584
 Name   = Pac-Man Rally
 Region = PAL-M5
-vuClampMode=2 // Fixes bad geometry.
+vuClampMode=3 // Fixes bad geometry.
```

Comment:
Very small formatting nitpick, adding space before and after equal sign for consistency.